looker_blueprints/download_dashboard.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import sys
+import time
+import os
+import shipyard_utils as shipyard
+from looker_sdk import models
+from looker_sdk.error import SDKError
+try:
+    import helpers
+except BaseException:
+    from . import helpers
+
+
+EXIT_CODE_LOOK_DASHBOARD_ERROR = 205
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--base-url', dest='base_url', required=True)
+    parser.add_argument('--client-id', dest='client_id', required=True)
+    parser.add_argument('--client-secret', dest='client_secret', required=True)
+    parser.add_argument('--dashboard-id', dest='dashboard_id', required=True)
+    parser.add_argument('--output-width', dest='output_width', required=True)
+    parser.add_argument('--output-height', dest='output_height', required=True)
+    parser.add_argument('--dest-file-name', dest='dest_file_name', required=True)
+    parser.add_argument('--dest-folder-name', dest='dest_folder_name', required=False)
+    parser.add_argument('--file-type', dest='file_type',
+                        choices=['pdf', 'png', 'jpg'],
+                        type=str.lower,
+                        required=True)
+    args = parser.parse_args()
+    return args
+
+
+def download_dashboard(sdk, dashboard_id, width, height, file_format):
+    """Download specified dashboard using ID
+    
+    Returns:
+        result: raw binary data of the dashboard
+    """
+    dash_id = dashboard_id
+    task = sdk.create_dashboard_render_task(
+        dash_id,
+        file_format,
+        models.CreateDashboardRenderTask(
+            dashboard_style='tiled',
+            dashboard_filters=None,
+        ),
+        width,
+        height,
+    )
+
+    if not (task and task.id):
+        print(
+            f'Could not create a render task for "{dashboard_id}"'
+        )
+        sys.exit(EXIT_CODE_LOOK_DASHBOARD_ERROR)
+    # poll the render task until it completes
+    elapsed = 0.0
+    delay = 0.5  # wait .5 seconds
+    while True:
+        poll = sdk.render_task(task.id)
+        if poll.status == "failure":
+            print(poll)
+            print(
+                f'Render failed for "{dashboard_id}"'
+            )
+            sys.exit(EXIT_CODE_LOOK_DASHBOARD_ERROR)
+        elif poll.status == "success":
+            break
+
+        time.sleep(delay)
+        elapsed += delay
+    print(f"Render task completed in {elapsed} seconds")
+
+    result = sdk.render_task_results(task.id)
```

Comment:
On line 61, is that the right sdk call? It seems more like a verb `render_task` than a getter. 

How fast is Looker expected to complete the render. Delaying for half a second per request seems like way too little time to wait. I would normally expect polling to be on the 5 to 10 second scale. Is there documentation stating you can poll this fast?

Reply:
Using the example provided by looker itself here:

https://github.com/looker-open-source/sdk-codegen/blob/main/examples/python/download_dashboard_pdf.py